RecoTracker/LSTCore/src/alpaka/TrackCandidate.h

```diff
@@ -456,6 +528,216 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     }
   };
 
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE void countSharedT5HitsAndFindUnmatched(QuintupletsConst quintuplets,
+                                                                        unsigned int candidateQuintupletIndex,
+                                                                        unsigned int baseQuintupletIndex,
```

Comment:
```suggestion
                                                                        unsigned int testT5,
                                                                        unsigned int refT5,
```
perhaps a chance for shorter code lines and similar clarity
(`candidateQuintuplet` also sounds like a T5 from a TC)